versions.tf

```
@@ -17,6 +17,6 @@
 terraform {
   required_version = "~> 0.12.6"
   required_providers {
-    google = "~> 2.15"
+    google = "~> 3.0.0"
```

Pin to major version, not patch.

```suggestion
    google = "~> 3.0"
```